src/middlewares/task.middleware.js

```diff
@@ -131,12 +139,22 @@ export default (store: any) => (next: any) => (action: any) => {
         additionalArgs.push('--', '--coverage');
       }
 
+      /* Bypasses 'Are you sure?' check when ejecting CRA
+       *
+       * TODO: add windows support
+       */
+      console.log('Task: ', action.task);
+      // const command =
+      //   project.type === 'create-react-app' && name === 'eject'
+      //     ? 'echo yes | npm'
+      //     : 'npm';
```

Comment:
Probably already on your radar, but we should kill these comments (both the commented-out code, but also the "TODO: add windows support" above :D)

Also the console.log was just for debugging, right? We should remove it as well.